### PR TITLE
Don't update suggested dependencies

### LINKED ISSUE #1749: `use_latest_dependencies()` shouldn't affect Suggests
Since those versions are not enforced

### PR DESCRIPTION
**Pull request by hadley**
Fixes #1749

Also a nice opportunity to use the new `proj_deps()` helper in the tests.

### CODE REVIEW

File: R/latest-dependencies.R [hunk 1]
```diff
@@ -1,7 +1,8 @@
 #' Use "latest" versions of all dependencies
 #'
-#' Pins minimum versions of dependencies to latest ones (as determined by `source`).
-#' Useful for the tidyverse package, but should otherwise be used with extreme care.
+#' Pins minimum versions of all `Import` and `Depends` dependencies to latest
```
Comment (jennybc) [inline R/latest-dependencies.R:3]:
```suggestion
#' Pins minimum versions of all `Imports` and `Depends` dependencies to latest
```

File: tests/testthat/test-latest-dependencies.R [hunk 2]
```diff
@@ -2,27 +2,39 @@ test_that("use_tidy_versions() specifies a version for dependencies", {
...
+
+test_that("use_tidy_versions() doesn't affect suggests", {
+  skip_on_cran()
+  withr::local_options(list(repos = c(CRAN = "https://cloud.r-project.org")))
+
+  create_local_package()
+  use_package("cli", "Suggests")
+
+  deps <- proj_deps()
+  expect_equal(deps$version[deps$package == "cli"], "*")
```
Comment (jennybc) [inline tests/testthat/test-latest-dependencies.R:25]: I don't see a call to `use_tidy_versions()` (or `use_latest_dependencies()`) here.

Comment (hadley) [reply tests/testthat/test-latest-dependencies.R:25]: Ooops 😆

**Review by jennybc — APPROVED** [review]
My parting thought is: the tests mention `use_tidy_versions()` but then actually call `use_latest_dependencies()`. Do we want to make that less puzzling?

**Pull request merged**